.github/workflows/ram_usage.yml

```diff
@@ -0,0 +1,94 @@
+name: SeqAn3 RAM-Usage
+
+on:
+  # Enables a manual trigger, may run on any branch
+  workflow_dispatch:
+    inputs:
+      compiler:
+        description: 'GCC version'
+        required: true
+        default: '11'
+      cxx_flags:
+        descriptions: 'CXXFLAGS to pass to compiler'
+
+env:
+  CMAKE_VERSION: 3.8.2
+  SEQAN3_NO_VERSION_CHECK: 1
+  TZ: Europe/Berlin
+
+defaults:
+  run:
+    shell: bash -exo pipefail {0}
+
+jobs:
+  build:
+    name: RAM-Usage gcc${{ github.event.inputs.compiler }}
+    runs-on: ubuntu-20.04
+    timeout-minutes: 300
+    steps:
+      - name: Checkout SeqAn3
+        uses: actions/checkout@v2
+        with:
+          path: seqan3
+          submodules: true
+
+      - name: Checkout SeqAn2
+        uses: actions/checkout@v2
+        with:
+          repository: seqan/seqan
+          ref: develop
+          path: seqan3/submodules/seqan
+
+      - name: Configure APT
+        continue-on-error: true
+        run: bash ./seqan3/.github/workflows/scripts/configure_apt.sh
+
+      - name: Install CMake
+        run: bash ./seqan3/.github/workflows/scripts/install_cmake.sh
+
+      - name: Install compiler g++-${{ github.event.inputs.compiler }}
+        run: sudo apt-get install --yes g++-${{ github.event.inputs.compiler }}
+
+      - name: Create gcc wrapper
+        run: |
+          echo -e '#!/bin/bash\n{ /usr/bin/time -v /usr/bin/g++-${{ github.event.inputs.compiler }} "$@"; }' > g++.sh
```

Comment:
Just a stupid question, would `-DCMAKE_CXX_COMPILER="/usr/bin/time -v /usr/bin/g++-${{ github.event.inputs.compiler }}"` work, or `-DCMAKE_CXX_COMPILER=g++-${{ github.event.inputs.compiler }} -DCMAKE_CXX_COMPILER_LAUNCHER="/usr/bin/time -v"` ?

Reply:
Maybe, but won't work with the redirect I need further down